pcdsdevices/epics/mps.py

```diff
@@ -101,3 +101,36 @@ def _fault_change(self, *args, **kwargs):
         """
         kwargs.pop('sub_type', None)
         self._run_subs(sub_type=self.SUB_FAULT_CH, **kwargs)
+
+
+def mps_factory(clsname, cls,  *args, mps_prefix=None, veto=False,  **kwargs):
```

Comment:
Any reason why mps_prefix is optional?

Reply:
Wanted to make sure we don't catch a stray `arg` and put it as `mps_prefix`?  But maybe I still don't avoid that... I'll change it to mandatory.